docs/streams/developer-guide/config-streams.html

```diff
@@ -416,68 +417,91 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
           <tr class="row-even"><td>poll.ms</td>
             <td>Low</td>
             <td colspan="2">The amount of time in milliseconds to block waiting for input.</td>
-            <td>100 milliseconds</td>
+            <td><code class="docutils literal"><span class="pre">100</span></code></td>
           </tr>
-          <tr class="row-odd"><td>rack.aware.assignment.tags</td>
-            <td>Medium</td>
+          <tr class="row-odd"><td>rack.aware.assignment.strategy</td>
```

Comment:
Adding missing parameters.

Also update importance level to LOW. (not sure why this should be MEDIUM)